src/pybind/rados/rados.pyx

```diff
@@ -294,7 +294,9 @@ cdef extern from "rados/librados.h" nogil:
     void rados_write_op_omap_rm_keys(rados_write_op_t write_op, const char * const* keys, size_t keys_len)
     void rados_write_op_omap_clear(rados_write_op_t write_op)
     void rados_write_op_set_flags(rados_write_op_t write_op, int flags)
-
+    void rados_write_op_setxattr(rados_write_op_t write_op, const char *name, const char *value, size_t value_len)
```

Comment:
can you update instances of `s/setxattr/set_xattr` and `s/rmxattr/rm_xattr`

Reply:
@ideepika updated, please review